styles.css

```diff
@@ -426,6 +429,33 @@ footer {
 .footer-text {
     align-items: flex-end;
 }
+.footer-sizing{
+    max-width: 15rem;
+    text-align: center;
+    padding: 1rem;
+}
+.agency-logo{
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     max-width: 20%;
+     height:auto;
+}
+i.fa-linkedin {
+    content: 'f08c';
```

Comment:
What does content: 'f08c' mean? 😮

Reply:
on font awesome its the uni code that makes the logo appear on the page with the right styling